include/clap/ext/draft/configurable-audio-ports.h

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include "../audio-ports.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// This extension lets the host configure the plugin's input and output audio ports.
+// This is a "push" approach to audio ports configuration.
+static CLAP_CONSTEXPR const char CLAP_EXT_CONFIGURABLE_AUDIO_PORTS[] =
+   "clap.configurable-audio-ports.draft0";
```

Comment:
Is `clap.audio-ports-config-info/draft-0` still needed with this extension existing? They technically cover different use cases, but `audio-ports-config-info` was added to better accommodate the VST3 bus arrangement setting protocol, and this new extension directly implements that protocol. If there's no demand for it anymore than it may be worth getting rid of. The changes to the ambisonics and surround extensions that accompanied this extension have also been removed.

Reply:
I also think we can remove it. @defiantnerd Do you agree?

Reply:
Let's continue this one in 1.1.9 so we allow some time.